src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponentSpec.scala

```diff
@@ -21,6 +21,7 @@ class ClusterComponentSpec extends TestComponent with FlatSpecLike with CommonTe
       googleProject = project,
       googleServiceAccount = googleServiceAccount,
       googleBucket = GcsBucketName("bucket1"),
+      machineConfig = MachineConfig(Some(0),Some(""), Some(500)),
```

Comment:
Maybe add some DB test cases with different `MachineConfig` values -- probably should try to write to every new column in unit tests.

Reply:
added below, thanks!